frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx

```diff
@@ -236,21 +236,23 @@ export const ParameterSettings = ({
       )}
 
       <Box mb="lg">
-        <SettingLabel>
+        <SettingLabel id="default-value-label">
           {t`Default value`}
           {parameter.required &&
             parameterHasNoDisplayValue(parameter.default) && (
-              <SettingLabelError>({t`required`})</SettingLabelError>
+              <SettingLabelError> ({t`required`})</SettingLabelError>
```

Comment:
I changed margin here for a whitespace. It arguably looks better this way (natural spacing).
But the primary reason is that user agent sees this text as `Default value(required)` - without the space.
It comes out when using `cy.findLabelByText`.